app/cdap/components/WrangleHome/Components/WidgetSVG/index.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Avatar } from '@material-ui/core';
+import ImageOutlined from '@material-ui/icons/ImageOutlined';
+import React from 'react';
+import styled from 'styled-components';
+import { IWidgetSrc } from 'components/WrangleHome/Components/WidgetSVG/types';
+
+const ImageContainer = styled.div`
+  font-size: 40px;
+`;
+
+export default function({ dataSrc }: IWidgetSrc) {
+  return dataSrc ? (
+    <Avatar
```

Comment:
This is essentially an icon right? Why not use https://mui.com/material-ui/icons/ IconSvg instead? Are the icons pngs?

Reply:
The icons are being fetched from the widget API. we are getting a string URL for the icon.
In case the widget does have the required icon, then we are using ImageOutlined from the material UI.